docs/process_to_manipulate_data.md

```diff
@@ -0,0 +1,44 @@
+### Process to manipulate data in production
+
+To manipulate data in production, the default workflow is to create a `sqitch` change, associated with `pgTap` tests that are run in our CI pipeline and tested in our `dev` and `test` environments as the application is deployed.
+
+In rare cases, the team needs to manipulate data directly in the database, without going through our usual guardrails. Examples of this are mostly around manipulation of private data. This repository is open-source, therefore we should not commit any private information.
+
+This should be considered a last resort. Updating data in production could lead to cascading failure if done without the proper guardrails, which our CI are providing.
+#### Important steps before you begin:
+
+- Get request/approval from Product Owner
+- Back up database before manipulating data
+- Pair program with another developer
+
+#### Manipulating the data
+
+Don't start in `prod`, always start in the `dev` namespace to practice and mitigate the chance of a failure in prod.
+
+1. Don't commit changes as superuser, change to the database user:
+
+`set role ccbc;`
+
+2. Always do the changes in a transaction:
+
+`begin;`
+
```

Comment:
```suggestion

At this point, you should see the following `psql` prompt: `ccbc=*>`. The `*` indicates that you are in a transaction, and the `>` is the normal user prompt (instead of `#` for a superuser)
```

It can be pretty stressful to do things without guardrails, lets' reassure the reader that they've run their steps properly